readme.md

```diff
@@ -59,19 +59,17 @@ cmake -S . -B build
 cmake --build build
 ```
 
-If you want to compile the GUI you can use the flag.
--DBUILD_GUI=ON
+The default build type is `Debug` if you want a `Release` build you can use the flag (on the first `cmake` command above): `-DCMAKE_BUILD_TYPE=Release`
 
-For rules support (requires pcre) use the flag.
--DHAVE_RULES=ON
+For release builds it is recommended that you use: `-DUSE_MATCHCOMPILER=ON`
 
-For release builds it is recommended that you use:
--DUSE_MATCHCOMPILER=ON
+If you want to compile the GUI you can use the flag: `-DBUILD_GUI=ON`
 
-For building the tests use the flag.
--DBUILD_TESTS=ON
+For rules support (requires pcre) use the flag: `-DHAVE_RULES=ON`
```

Comment:
Maybe we can use this change to explain what rules are and if one should enable it or not.

Reply:
Its unrelated to my change or my PR now.